### DISCUSSION TITLE
ROR Curve Not Showing During Roast

### DISCUSSION THREAD

**Discussion by McDoogle523** [artisan-roaster-scope/artisan#623] [Q&A] [answered]
Hello! This is an SOS. I've been absolutely wracking my brains out but I just can't seem to get Artisan to do what I want. I've scoured the Quick-Start guide, asked Home-Barista, and fiddled with things myself, but I can't get the RoR curve to appear during my roasts. It shows up after my roast is already completed.

[Here are some screenshots of my settings](https://imgur.com/a/izK6gAk). What am I doing wrong? 

Edit: I am hitting Charge at the beginning of the roast, forgot to mention that.

**Comment by roasterdave** [comment, marked as answer]
You must mark a CHARGE event before the RoR curves are shown live.

**Reply by McDoogle523** [reply]
Thanks for your response! I foolishly omitted this information but I am in fact hitting Charge and still having this issue.

**Reply by roasterdave** [reply]
Please post your .aset settings file file (menu Help > Save Settings) and a .alog file for a profile where this happened during the roast.  This will help to see what is wrong.  

Please note that files must be attached by using your browser, they can't be attached using email.  Add a .txt extension or zip the files before uploading. Otherwise you will receive a "Not a supported file type" error on uploading.  Thanks.

**Reply by McDoogle523** [reply]
[Artisan.zip](https://github.com/artisan-roaster-scope/artisan/files/6588876/Artisan.zip)

Files attached. Thanks for your help!

**Reply by roasterdave** [reply]
Thanks.  Please go to Config > Axes.  Under delta Axis uncheck Auto BT.  Then set delta Axis Max to 40 and step to 10.  These numbers were chosen to fit the profile you posted assuming it is typical for your roaster.  Then click OK.

When Auto is checked it is possible for Artisan to set your delta Axis Max to 1 at the start of a roast.  When that happens your delta curves are being recorded but they can't be seen because they are way above the maximum.  When the roast is finished, Artisan recalculates the auto axis and you see the curves.  

This problem will be fixed in the next release.  A continuous build with this fix is available [here](https://github.com/artisan-roaster-scope/artisan/releases/tag/continuous) if you want to test it.

![image](https://user-images.githubusercontent.com/30009312/120589846-5f186880-c3ee-11eb-96bd-235cbe619463.png)

**Reply by McDoogle523** [reply]
This seems to have done it! Thanks again for your help.

**Comment by CarefreeBuzzBuzz** [comment]
[FAQ #7 under Recording](https://artisan-scope.org/help/)

**Reply by CarefreeBuzzBuzz** [reply]
Also added a line on Curves page of the QSG.  

First you decide which ROR curves and LCD’s you want to display. **NOTE you wont see your ROR curves without pushing the CHARGE button during the roast.**

**Reply by McDoogle523** [reply]
Thanks for your response! I foolishly omitted this information but I am in fact hitting Charge and still having this issue. I think timing might be the issue - just to confirm, if I don't wait until 5 sample periods pass (so 15 seconds by default), I won't see a curve during my roast at all? I've been hitting Charge immediately upon beginning the roast (I have a fluid bed so my beans are already in the roaster before I engage the heating elements, so they're "precharged" in a way).

**Reply by CarefreeBuzzBuzz** [reply]
The 15 seconds really only applies to Auto Charge.  You'll need to give us more info as if you are getting readings and you have confirmed a charge event it should work.  Do you see a Charge event on the Events tab of Roast Properties?

**Reply by McDoogle523** [reply]
![Events Tab](https://user-images.githubusercontent.com/26385941/120583398-24b5c800-c3fc-11eb-945a-a537e9ec5857.JPG)

Here's what my default Events tab looks like.

**Reply by poundy** [reply]
is this a SAVED profile? If so, none of your marks have been recorded!
Edit: I take that back. Look on the DATA tab and look to confirm that a specific time entry has CHARGE marked in the first column

**Reply by McDoogle523** [reply]
It's strange, this roast clearly has the Charge marked but it's not showing up in the roast properties at all.

Edit: Wait, sorry, it is here. I'm coming off RoastTime so the concept of Charge not being the very first thing is still new and strange to me. What I wouldn't do for that level of plug n play right now...

![Charge](https://user-images.githubusercontent.com/26385941/120584695-6fd0da80-c3fe-11eb-95dc-fb9274271cba.JPG)
![Data Tab](https://user-images.githubusercontent.com/26385941/120584545-3009f300-c3fe-11eb-8441-f0aad327b815.JPG)
![Events 02](https://user-images.githubusercontent.com/26385941/120584547-30a28980-c3fe-11eb-9b72-03ae22c75f15.JPG)
![Roast](https://user-images.githubusercontent.com/26385941/120584548-30a28980-c3fe-11eb-9ffc-ca62052594cc.JPG)
